src/main/java/org/cactoos/RepeatedCallable.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017-2020 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos;
+
+import java.util.concurrent.Callable;
+
+/**
+ * Callable that runs repeatedly for a number of times.
+ *
+ * @param <T> Type of output
+ * @since 0.49.2
+ */
+public final class RepeatedCallable<T> implements Callable<T> {
+
+    /**
+     * Original callable.
+     */
+    private final Callable<T> callable;
+
+    /**
+     * How many times to run.
+     */
+    private final int times;
+
+    /**
+     * Ctor.
+     *
+     * <p>If {@code max} is equal or less than zero {@link #call()} will return
+     * an exception.</p>
+     *
+     * @param cllbl Callable to repeat.
+     * @param max How many times.
+     */
+    public RepeatedCallable(final Callable<T> cllbl, final int max) {
+        this.callable = cllbl;
+        this.times = max;
+    }
+
+    @Override
+    public T call() throws Exception {
+        if (this.times <= 0) {
+            throw new IllegalArgumentException(
+                "The number of repetitions must be at least 1"
+            );
+        }
+        T result = null;
+        for (int idx = 0; idx < this.times; ++idx) {
+            result = this.callable.call();
+        }
+        return result;
```

Comment:
@fabriciofx instead of reimplementing this here, reuse `Repeated` here and for example call it with `true` and a `Func` that calls the callable and returns `true` also. just instantiate it and call it from here

Reply:
@victornoel I tried to do this before, but I couldn't. But now I did! Thanks!